components/newsroom/Newsroom.tsx

```diff
@@ -78,7 +84,13 @@ export default function Newsroom() {
           </div>
           <div className='w-full px-2 md:w-1/2 md:pl-4 md:pr-0'>
             <div className='mx-auto mt-8 w-full rounded-xl shadow-md md:mt-0' data-testid='Newsroom-Twitter'>
-              <TwitterTimelineEmbed sourceType='profile' screenName='AsyncAPISpec' options={{ tweetLimit: '2' }} />
+            {!isLoaded && <p> Loading Tweets... </p>}
```

Comment:
There is a [loader component](https://github.com/asyncapi/website/blob/master/components%2FLoader.tsx) present in the codebase. You should use that here.